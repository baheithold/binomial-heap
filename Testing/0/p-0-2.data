8 5 303 ; 0 9 477 ; 4 3 41 ; 0 8 424 ; 0 7 332 ; 
8 3 487 ; 1 6 12 ; 4 3 155 ; 1 6 78 ; 8 1 237 ; 
5 5 484 ; 5 4 184 ; 0 8 174 ; 9 2 258 ; 6 4 348 ; 
2 1 101 ; 0 6 309 ; 3 1 332 ; 1 9 407 ; 3 9 40 ; 
0 5 18 ; 4 9 366 ; 4 3 131 ; 7 3 213 ; 3 3 242 ; 
7 4 81 ; 3 6 455 ; 4 8 65 ; 5 7 20 ; 4 8 181 ; 
1 6 86 ; 0 4 38 ; 5 5 15 ; 8 1 409 ; 2 5 498 ; 
8 6 262 ; 4 3 39 ; 5 8 342 ; 2 0 178 ; 0 2 273 ; 
9 0 70 ; 3 9 51 ; 1 4 254 ; 2 5 499 ; 7 4 250 ; 
6 6 354 ; 9 7 412 ; 0 1 439 ; 0 9 266 ; 4 4 448 ; 
5 6 231 ; 6 0 280 ; 2 4 165 ; 7 5 166 ; 3 7 200 ; 
1 1 158 ; 8 6 176 ; 9 7 5 ; 0 3 267 ; 7 3 130 ; 
3 4 274 ; 3 6 491 ; 9 2 467 ; 7 0 421 ; 4 1 411 ; 
6 1 434 ; 4 9 106 ; 1 6 316 ; 6 4 408 ; 9 4 58 ; 
6 0 245 ; 4 2 391 ; 8 1 46 ; 9 3 466 ; 2 4 205 ; 
5 8 309 ; 4 6 337 ; 2 5 201 ; 6 3 192 ; 2 3 280 ; 
3 9 62 ; 5 0 268 ; 6 6 382 ; 1 4 423 ; 5 9 352 ; 
6 1 426 ; 4 8 303 ; 9 0 159 ; 2 4 351 ; 6 5 148 ; 
7 7 382 ; 9 7 125 ; 7 1 72 ; 4 4 191 ; 3 8 402 ; 
6 4 470 ; 5 9 328 ; 5 8 82 ; 1 0 254 ; 8 0 398 ; 

