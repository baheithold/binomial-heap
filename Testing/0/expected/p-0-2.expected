0: 0
1: 3(0)21 9(0)56
2: 2(9)55 5(9)66 8(3)161
3: 1(2)61 6(5)65
4: 4(6)33 7(1)5
weight: 523
